app/jarkur/main.c

```diff
@@ -0,0 +1,56 @@
+#include <stdio.h>
```

Comment:
This is not part of task.
`main` from `tests.c` shoud be enough for testing purposes (eventually some online compiler or proper .gitignore file)